frontend/src/app/core/services/project.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { ProjectDto } from '../../models/projects/project-dto';
+
+import { HttpInternalService } from './http-internal.service';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class ProjectService {
+    private readonly projectsApiUrl = '/api/project';
```

Comment:
the first part of Url should be here?

like `env.CoreUrl`

Reply:
> the first part of Url should be here?
> 
> like `env.CoreUrl`

If I'm not mistaken, no. I'll look again tomorrow.

Reply:
> Collaborator

Do you mean to add /api to env.coreUrl like this:

export const environment = {
    production: false,
    coreUrl: 'http://localhost:5050',
    notifierUrl: 'http://localhost:5070',
};

Reply:
yes, `${environment.coreUrl}/api/project`